<ui version="4.0" >
 <class>ProgressWidget</class>
 <widget class="QWidget" name="ProgressWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>8</number>
   </property>
   <property name="leftMargin" >
    <number>16</number>
   </property>
   <property name="topMargin" >
    <number>16</number>
   </property>
   <property name="rightMargin" >
    <number>16</number>
   </property>
   <property name="bottomMargin" >
    <number>16</number>
   </property>
   <item>
    <widget class="QLabel" name="topLabel" >
     <property name="text" >
      <string>Task in progress</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar" >
     <property name="value" >
      <number>0</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="bottomLabel" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
